### PR TITLE
Add a whole bunch of new status commands

### PR DESCRIPTION
**Pull request by lyxal**

### CODE REVIEW

File: src/commands.js [hunk 1]
```diff
@@ -226,7 +226,7 @@ export default {
 
         if (
             content.match(
-                /^(status|((lol )?(yo)?u good( (there )?(my )?(epic )?(bro|dude|sis|buddy|mate|m8|gamer)?)?\?*))$/i
+                /^((what'?s your )status|((lol )?(yo)?u good( (there )?(my )?(epic )?(bro|dude|sis|buddy|mate|m8|gamer)?)?\?*)|(what|how) (is our beep boop|are you|the bot) doin[g']?\??|how are you\??|do you( is| are| done)* do\??|(beep boop|bot)( doing)? ok(ay)?\??)$/i
```
Comment (cgccuser) [inline src/commands.js]:
```suggestion
                /^((what'?s your )?status|((lol )?(yo)?u good( (there )?(my )?(epic )?(bro|dude|sis|buddy|mate|m8|gamer)?)?\?*)|(what|how) (is our beep boop|are you|the bot) doin[g']?\??|how are you\??|do you( is| are| done)* do\??|(beep boop|bot)( doing)? ok(ay)?\??)$/i
```

Comment (cgccuser) [reply src/commands.js]: Btw did you test this on regex101.com or something?

**Review by cgccuser — CHANGES_REQUESTED** [review]

File: src/commands.js (hunk 1 above)
Comment (lyxal) [reply src/commands.js]: yes

**Review by cgccuser — APPROVED** [review]
LGTM. [regex101](https://regex101.com/r/7swDza/1) for anyone wanting to check it out themselves.

**Pull request merged**